### PR TITLE
Contributing documentation: add WWW-merge

### PR DESCRIPTION
**Pull request by sbesson**
The main purpose of this PR is to add the CI job allowing to integrate and review the PRs opened against the OME website. The page about the Jekyll websites is also modified to include the new website.

To test this PR, review the new WWW-merge entry.

### CODE REVIEW

**Comment by kennethgillen** [comment]
@sbesson  - according to https://ci.openmicroscopy.org/job/CONTRIBUTING-merge-docs/1125/console this is in the merge build for CONTRIB docs - but I don't see it on http://docs-staging.openmicroscopy.org/contributing/ci-docs.html. 

Am I missing something?

**Comment by sbesson** [comment]
@kennethgillen this is where the deployment/URL can get tricky. The built documentation including the PRs for review is currently only available from the Jenkins workspace or as an archive of the build. The URL you mentioned points to the same static content as https://docs.openmicroscopy.org but from `ome-web-staging`

**Comment by kennethgillen** [comment]
Okay, thanks. 👍

File: contributing/ci-docs.txt [hunk 1]
```diff
@@ -298,26 +271,33 @@ located in the ome-model repository and is built from the master branch.
...
+:doc:`OME Jekyll websites <jekyll>`.
 
 .. glossary::
 
+	:jenkinsjob:`WWW-merge`
+
+		This job is used to review the PRs opened against the master branch of
+		https://github.com/openmicroscopy/www.openmicroscopy.org
+
+		#. |merge| and push the branch to https://github.com/snoopycrimecop/www.openmicroscopy.org/tree/gh-pages
```
Comment (kennethgillen) [inline contributing/ci-docs.txt]: and "pushes" ?

**Review by kennethgillen — COMMENTED** [review]
[this was supposed to be "request changes", doing below]

**Review by kennethgillen — CHANGES_REQUESTED** [review]
https://github.com/openmicroscopy/ome-documentation/pull/1719#pullrequestreview-51209245

**Comment by kennethgillen** [comment]
👍  good to merge.

**Review by kennethgillen — APPROVED** [review]

File: contributing/ci-docs.txt [hunk 2]
```diff
@@ -298,26 +271,33 @@ located in the ome-model repository and is built from the master branch.
 		#. |sphinxbuild|
 		#. |linkcheck|
 
-OME Help
-^^^^^^^^
+Jekyll websites
+^^^^^^^^^^^^^^^
 
-The repository for the OME help is https://github.com/openmicroscopy/ome-help.
+The following set of jobs are used to review or publish the content of the
```
Comment (jburel) [inline contributing/ci-docs.txt]: typo it should be ``is used`` instead of ``are used``

File: contributing/ci-docs.txt [hunk 3]
```diff
@@ -298,26 +271,33 @@ located in the ome-model repository and is built from the master branch.
...
 
 .. glossary::
 
+	:jenkinsjob:`WWW-merge`
+
+		This job is used to review the PRs opened against the master branch of
+		https://github.com/openmicroscopy/www.openmicroscopy.org
+
+		#. |merge| and pushes the branch to https://github.com/snoopycrimecop/www.openmicroscopy.org/tree/gh-pages
+		#. The GitHub pages deploys the staging website content under http://snoopycrimecop.github.io/www.openmicroscopy.org/
```
Comment (jburel) [inline contributing/ci-docs.txt]: deploy

**Comment by sbesson** [comment]
@jburel pushed

**Comment by jburel** [comment]
Looks good merging

**Pull request merged**